.appveyor.yml

```diff
@@ -0,0 +1,43 @@
+environment:
+  matrix:
+  - julia_version: 0.7
```

Comment:
I think 1.0 only is fine by now

Reply:
Well, I matched what you declare in `REQUIRE`... I think the two should match, else you make a promise that is not verified. So, which is it -- drop 0.7 here and in `REQUIRE`, or keep it in both?

Reply:
Good point, we should bump the REQUIRE.

Reply:
The drawback of that is that if any other parts of OSCAR will require 0.7 for some time still, they won't be usable together with Polymake.jl... Is there a downside to keeping 0.7 support in for now?

Perhaps you are more confident than me that all parts of OSCAR will "soon" support Julia 1.0?

Reply:
If there is a need to still support 0.7, I think we can also do this. But things like `AbstractAlgebra.jl` doesn't work with 0.7 (there is a compiler bug in 0.7) so I don't see how OSCAR will run under 0.7 :)

Reply:
Well, I am forced to run under Julia master branch, because 1.0 is missing crucial infra structure for GAPJulia, so I guess you are right. I was mostly thinking about Singular.jl, but I guess we'll simply have to wait for them to support 1.0 / 1.1.

Will drop 0.7 from here and REQUIRE